### PR TITLE
cpulimit: add page

### PR DESCRIPTION
**Pull request by sbrl**
Follow-on from #2857, since it wouldn't let me edit the original PR

### CODE REVIEW

File: pages/linux/cpulimit.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# cpulimit
+
+> A tool to throttle the CPU usage of other processes.
+> Homepage: <http://limitcpu.sourceforge.net/>.
```
Comment (mebeim) [inline pages/linux/cpulimit.md]:
```suggestion
> More information: <http://cpulimit.sourceforge.net/>.
```

File: pages/linux/cpulimit.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+- Limit an existing process with PID 1234 to only use 25% of the CPU:
+
+`cpulimit -p {{1234}} -l {{25%}}`
+
+- Limit an existing program by its executable name instead of PID number:
```
Comment (agnivade) [inline pages/linux/cpulimit.md]: I think - "instead of PID number" can be removed.

File: pages/linux/cpulimit.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+- Launch a given program and limit it to only use 50% of the CPU:
+
+`cpulimit -l {{50}} {{program}}`
+
+- Launch a program, limit its CPU usage to 50% and run cpulimit in the background:
+
+`cpulimit -l {{50}} -b {{program}}`
+
+- Launch a program and kill its process if the program's CPU usage goes over 50%:
```
Comment (agnivade) [inline pages/linux/cpulimit.md]: No need to say "Launch a program" after the first 2 times. Let us just start with "Kill the process .."

File: pages/linux/cpulimit.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+`cpulimit -l 50 -k {{program}}`
+
+- Launch a program and throttle both it and its child processes so that none go about 25% CPU:
```
Comment (agnivade) [inline pages/linux/cpulimit.md]: Is there an expanded flag for `-m` ? It does not seem to match with throttle. Also where are these flags documented ? I can't seem to find those in http://cpulimit.sourceforge.net/.

**Comment by sbrl** [comment]
Updated, @agnivade

**Comment by agnivade** [comment]
I wanted to check expanded flags for only `-m`, but this is fine too :slightly_smiling_face:

**Comment by sbrl** [comment]
Oops! Missed that one. Done

**Pull request merged**